docs/source/developer/initial_setup.rst

```diff
@@ -0,0 +1,81 @@
+**************************************
+Setting Up CMakePPLang for Development
+**************************************
+
+This page contains information to help developers new to CMakePPLang set up
+the project to begin development.
+
+Downloading the Source Code
+===========================
+
+During development, the latest version of CMakePPLang should be used.
+CMakePPLang is `hosted on GitHub <CMakePPLang_src_>`_ and can be cloned to
```

Comment:
```suggestion
CMakePPLang is `hosted on GitHub <CMakePPLang_src_>`__ and can be cloned to
```

Unless you're going to do:

```.rst
:ref:`hosted on GitHub`
```

somewhere put a second underscore after your links. This stops Sphinx from throwing warnings in case you ever use "hosted on GitHub" again as link text.